src/main/java/com/elytradev/concrete/resgen/MissingRealpackException.java

```diff
@@ -0,0 +1,10 @@
+package com.elytradev.concrete.resgen;
+
+/**
+ * Thrown when a ConcreteResourcePack can't find a valid fallback pack to use.
+ */
+public class MissingRealpackException extends RuntimeException {
```

Comment:
Still required as we need to store the real pack, therefore if it's not present we have to scream about it.